index.html

```diff
@@ -2,12 +2,82 @@
 <html>
 <head>
 <link href="style.css" rel="stylesheet" type="text/css">
+<script src="https://kit.fontawesome.com/173db1d6d6.js" crossorigin="anonymous"></script>
 <title>Portafolio</title>
 </head>
 <body>
 
-<h1>This is a Heading</h1>
-<p>This is a paragraph.</p>
+    <section>
+
+        <div class="wrapper">
+            <header class="toolbar">
+                <nav>            
+                    <a href="#">
+                    <li class="item">Edahí Gutiérrez </li> </a>        
+                    <li id="item-right" class="item">&#9776;</li>                                
+                </nav>
+            </header>
+            
+        
+            <h1>Hey There. <br>
+                I’m Edahí</h1>
+        
+            <h2 id="intro">I’am a Software Developer</h2>   
+        
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don’t hestiate to contact me. </p>
+        
+            <div class="social-group">
+                <ul>
+                    <li class="social">
+                        <a href="http://github.com/edahigure">
+                            <i class="fa fa-github"></i></a>
+                    </li>
+                    <li class="social">
+                        <a href="https://linkedin.com/in/edahigure">
+                            <i class="fa-brands fa-linkedin-in"></i></a>
+                    </li>
+                    <li class="social">
+                        <a href="https://twitter.com/edahigure">
+                            <i class="fa-brands fa-twitter"></i></a>
+                    </li>
+            
+                    <li class="social">
+                        <a href="#">
+                            <i class="fa-brands fa-angellist"></i></a>
+                    </li>
+                    
+                    <li class="social">        
+                        <a href="#">
+                            <i class="fa-solid fa-m"></i>
+                        </a>
+                
+                    </li>
+        
+                    <li class="image"></li>
+        
+                </ul>
+                
+        
+                
+        
+            </div>
+            
+        
+            
+            
+        
+            
+        </div>
+
+
+    </section>
```

Comment:
As requested by the previous reviewer, you should wrap your code into sections. Currently, you are wrapping all your code into one big section, which isn't semantically accurate. For this project, you can have your header (which is correctly done by you) and a section that contains your personal information/introduction. It could look like this:
<_header_>
<_nav_>
<_/nav_>
<_/header_>

<_section_>
INFORMATION ABOUT YOU
<_/section_>

<_another section_>
ANOTHER PART (think like literally another section) OF THE PROJECT (maybe your cards, contact or something else)

This will make your code semantically accurate, more maintainable and easier to style 😃.

Reply:
removed extra new lines